src/Browscap/Data/Expander.php

```diff
@@ -43,6 +43,11 @@ class Expander
     private $logger = null;
 
     /**
+     * @var array
```

Comment:
Could turn this into an object I suppose, and type-hint with that.  My only concern would be the added overhead to do that compared to a simple associative array here, as this id is built for every pattern that is encountered in the useragent resource files.